src/main/java/com/avispl/symphony/dal/communicator/barco/e2gen2/BarcoE2Communicator.java

```diff
@@ -540,7 +551,7 @@ private void getPresetFeedBack(Map<String, String> stats, List<AdvancedControlla
 		for (int i = 0; i < response.size(); i++) {
 			JsonNode responseNodeName = response.get(i);
 			if (responseNodeName != null && responseNodeName.get(BarcoE2Constant.NAME) != null) {
```

Comment:
responseNodeName.get(BarcoE2Constant.NAME) may be extracted into a variable so there won't be need to fetch it multiple times

Reply:
Fixed